packages/kbn-avc-banner/src/index.tsx

```diff
@@ -14,6 +14,13 @@ import { FormattedMessage } from '@kbn/i18n-react';
 import { useKibana } from '@kbn/kibana-react-plugin/public';
 import avcBannerBackground from './avc_banner_background.svg';
 
+// Logic to hide banner at EOY 2024
+export const useIsStillYear2024 = () => {
+  const year2025 = new Date('January 01, 2025 00:00:00');
+  const today = new Date(Date.now());
+  return today < year2025;
```

Comment:
can you memoize this with `useMemo()`

Also - please add a return type (boolean) to the hook

Reply:
This function isn't inside another component, so useMemo() wouldn't work here. I thought the date calculation would be simple enough where memoizing doesn't really make that much of a difference. Is this actually not the case?